docs/ebi-wrangler-onboarding.md

```diff
@@ -186,6 +187,15 @@ We are required to record our time working at the EBI in timesheets. A couple of
 - Try to remember to submit your timesheet each week as a minimum, or strive for daily to ensure you're always recording your time accurately while it's fresh in your mind
 - The hard deadline for submitting a timesheet is by the last day of each calendar month to ensure your supervisor has enough time to review and approve
 
+#### Leaves of Absence
+
+1. request from line manager on SAP
+2. advance notice required is equal to the length of vacation
+3. mark your personal calendar as _busy_ or our of office so people who try to invite you to a meeting at the specific time you would be off can notice you would be away.
+4. add an entry to the AIT Calendar with a general comment about you being away, e.g. _John AL_. This way, team mates will notice you are away in the standup session. ![image](https://user-images.githubusercontent.com/927145/193549987-f71dbd17-4aec-400e-9405-8a24cd3fa4bd.png)
+
+5. notify the #hca channel: _I plan to be on vacation on  dd/mm_
```

Comment:
It's usually a good practice to do this before actually requesting the leave; while it will be very rare that holidays can't be given on those days, there is the possibility, and first asking then requesting is a better practice IMO